Tests/TensorFlowTests/SequentialTests.swift

```diff
@@ -58,6 +59,7 @@ final class SequentialTests: XCTestCase {
                 amsgrad.update(&model, along: 𝛁model)
                 adagrad.update(&model, along: 𝛁model)
                 adadelta.update(&model, along: 𝛁model)
+                radam.update(&model, along: 𝛁model)
             }
         }
         XCTAssertEqual(model.inferring(from: [[0, 0], [0, 1], [1, 0], [1, 1]]),
```

Comment:
I wonder if you ran this test locally, after adding `RAdam`? The expected results of `model.inferring` should be updated.

Reply:
+1, do check if build and test pass locally.

Reply:
The test did pass locally. Will run it again and get back to you.

Reply:
Test is failing now - result's going to NaN. Anyway I can debug this ?

Reply:
> Test is failing now - result's going to NaN. Anyway I can debug this ?

I'd recommend writing a standalone unit test for rectified Adam. This will help isolate the issue.

Here's a [simple reference test for rectified Adam](https://github.com/tensorflow/addons/blob/b4cea8189724f88663637daa1df82d035d97cba4/tensorflow_addons/optimizers/rectified_adam_test.py#L29) - would you like to port it to a new file `Tests/TensorFlow/OptimizerTests.swift`?

Reply:
Yeah, I will be porting it to a new file like you've suggested. I've found the issue, will make the changes and submit `OptimizerTests` as a separate PR.

Reply:
Looks like the test is still failing:
```
XCTAssertEqual failed: ("[[0.5567076],
 [0.5567076],
 [0.5567076],
 [0.5567076]]") is not equal to ("[[0.5053005],
 [0.5053005],
 [0.5053005],
 [0.5053005]]")
```